### ISSUE TITLE
Not sure what the internal problem is (was working fine then crashed - CMD not working)

### ISSUE THREAD

**Issue by greenelation** [spyder-ide/spyder#13951]
### What steps will reproduce the problem?

CMD has never worked when running CMD of python pg

### Traceback
```python-traceback
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\editor\utils\autosave.py", line 116, in do_autosave
    stack.autosave.autosave_all()
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\editor\utils\autosave.py", line 375, in autosave_all
    self.maybe_autosave(index)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\editor\utils\autosave.py", line 335, in maybe_autosave
    orig_hash = self.file_hashes[orig_filename]
KeyError: 'C:\\Users\\tlew\\.spyder-py3\\autosave\\P5_Family.py'
```

## Versions

* Spyder version: 4.0.1 
* Python version: 3.7.6
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```
atomicwrites >=1.2.0         :  1.3.0 (OK)
chardet >=2.0.0              :  3.0.4 (OK)
cloudpickle >=0.5.0          :  1.3.0 (OK)
diff_match_patch >=20181111  :  20181111 (OK)
intervaltree                 :  None (OK)
IPython >=4.0                :  7.12.0 (OK)
jedi =0.14.1                 :  0.14.1 (OK)
nbconvert >=4.0              :  5.6.1 (OK)
numpydoc >=0.6.0             :  0.9.2 (OK)
pexpect >=4.4.0              :  4.8.0 (OK)
pickleshare >=0.4            :  0.7.5 (OK)
psutil >=0.3                 :  5.6.7 (OK)
pygments >=2.0               :  2.5.2 (OK)
pylint >=0.25                :  2.4.4 (OK)
pyls >=0.31.2;<0.32.0        :  0.31.7 (OK)
zmq >=17                     :  18.1.1 (OK)
qdarkstyle >=2.7             :  2.8 (OK)
qtawesome >=0.5.7            :  0.6.1 (OK)
qtconsole >=4.6.0            :  4.6.0 (OK)
qtpy >=1.5.0                 :  1.9.0 (OK)
rtree >=0.8.3                :  0.9.3 (OK)
sphinx >=0.6.6               :  2.4.0 (OK)
spyder_kernels >=1.8.1;<2.0.0:  1.8.1 (OK)
watchdog                     :  None (OK)
cython >=0.21                :  0.29.15 (OK)
matplotlib >=2.0.0           :  3.1.3 (OK)
numpy >=1.7                  :  1.18.1 (OK)
pandas >=0.13.1              :  1.0.1 (OK)
scipy >=0.17.0               :  1.4.1 (OK)
sympy >=0.7.3                :  1.5.1 (OK)
```

**Comment by steff456** [comment]
Hi @greenelation,

Can you please give us a detailed explanation of how we can reproduce this issue? Also it will be really helpful if you are able to take screenshots or a gif when the error appears.

**Comment by greenelation** [comment]
Hi Steff, Thanks for responding. I am actually a little confused. I remember working with Pythan via Spider.
Then I got this message to log to github. ATM Things are working fine in Spider.

**Comment by ccordoba12** [comment]
> ATM Things are working fine in Spider.

That doesn't mean this error won't appear again. Since this error is fixed in our latest version (4.1.5) and you're using an old one (4.0.1), please update by closing Spyder, opening the Anaconda Prompt and running there

    conda update anaconda
    conda install spyder=4.1.5

**Comment by ccordoba12** [comment]
This problem is fixed in our latest version available in Anaconda (5.3.3). Please update by closing Spyder, opening the Anaconda Prompt and running there the commands mentioned in [our documentation](http://docs.spyder-ide.org/current/installation.html#updating-with-anaconda).
